.travis.yml

```diff
@@ -1,2 +1,6 @@
+notifications:
+  email: false
+
 rvm:
-  - 2.1.0
+  - 2.1.8
+  - 2.2.4
```

Comment:
Hmmm, let's do 2.1.2 (for prod compat), latest 2.1.x, latest 2.2.x and add 2.3.x and eval the travis outcomes.  If it borks, modify, otherwise it's good.  The point about 2.1.x diffs makes sense and, if we upgrade prod, drop the 2.1.2 in favor of whatever is on prod.  Assuming prod/stage are in sync, but, if not, also add stage version.  (Last I checked, all the sdr -dev, -stage and -prod are in sync on ruby version.)